lms/static/js/edxnotes/models/note.js

```diff
@@ -46,7 +46,7 @@ define(['backbone', 'js/edxnotes/utils/utils', 'underscore.string'], function (B
             var message = this.get('quote');
 
             if (!this.get('is_expanded') && this.get('show_link')) {
-                message = _.str.prune(message, this.textSize);
+                message = _str.prune(message, this.textSize);
```

Comment:
I'd prefer that you import Underscore and then use the standard notation here. Is there a reason why that wouldn't work?